test/config/100-kafka-ephemeral-triple-2.6.0.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: kafka.strimzi.io/v1beta1
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    version: 2.6.0
+    replicas: 1
+    listeners:
+      - name: plain
+        port: 9092
+        type: internal
+        tls: false
+      - name: tls
+        port: 9093
+        type: internal
+        tls: true
+    config:
+      offsets.topic.replication.factor: 3
```

Comment:
A cluster composed of 1 replica but the offsets replication factor is 3.
Is it valid?

Reply:
`offsets.topic.replication.factor`
```
The replication factor for the offsets topic (set higher to ensure availability). Internal topic creation will fail until the cluster size meets this replication factor requirement.
```

Reply:
stupid typo - I've updated @pierDipi